src/test-apps/happy/lib/WeaveState.py

```diff
@@ -238,17 +238,32 @@ def getWeaveGlobalPrefix(self, fabric_id):
 
         return global_prefix
 
-    def typeToWeaveSubnet(self, type):
-        if type == "wifi":
+    def typeToWeaveSubnet(self, interface_type):
```

Comment:
I changed type to interface_type
type is Python key word, we should avoid to use it in our code.

Reply:
Thanks; much improved!